### PR TITLE
[WIP] Execution chart + Sorting

### PR DESCRIPTION
**Pull request by tagantroy**

### CODE REVIEW

File: fork-runner/src/main/java/com/shazam/fork/ForkConfiguration.java [hunk 1]
```diff
@@ -104,6 +104,17 @@
      */
     public String includedAnnotation;
 
+    /**
+     * TODO
+     */
```
Comment (cdsap) [inline fork-runner/src/main/java/com/shazam/fork/ForkConfiguration.java]: ?

File: fork-runner/src/main/java/com/shazam/fork/sorting/QueueProvider.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+public class QueueProvider {
+
+    private final Configuration configuration;
+
+    public QueueProvider(Configuration configuration) {
+        this.configuration = configuration;
+    }
+
+    public Queue<TestCaseEvent> create() {
+        if (configuration.getSortingStrategy().statistics != null) {
```
Comment (cdsap) [inline fork-runner/src/main/java/com/shazam/fork/sorting/QueueProvider.java:19]: `sortingStrategy` can be null?

**Review by antonbatishchev — APPROVED** [review]
Huh, we need DI :)

File: fork-runner/src/main/java/com/shazam/fork/summary/executiontimeline/Measure.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    public ExecutionStats getExecutionStats() {
+        return executionStats;
+    }
+
+    public List<Data> getData() {
+        return data;
+    }
+}
+
+
```
Comment (cdsap) [inline fork-runner/src/main/java/com/shazam/fork/summary/executiontimeline/Measure.java]: line

File: fork-stat-common/build.gradle [hunk 4]
```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java-library'
+apply from: '../build-scripts/gradle-mvn-push.gradle'
```
Comment (cdsap) [inline fork-stat-common/build.gradle:2]: $ROOT_DIR?

**Review by cdsap — COMMENTED** [review]
small things, looks great

**Review by cdsap — APPROVED** [review]

**Pull request merged**